src/service/resources-impl.js

```diff
@@ -1304,7 +1181,37 @@ export class ResourcesImpl {
     // Phase 2: Remeasure if there were any relayouts. Unfortunately, currently
     // there's no way to optimize this. All reads happen here.
     let toUnload;
-    if (
+    if (this.intersectionObserver_) {
+      // The IntersectionObserver variant for phase 2 is just a simplification
+      // that ignores `relayoutTop` and `elementsThatScrolled`.
+      for (let i = 0; i < this.resources_.length; i++) {
+        const r = this.resources_[i];
+        if (r.hasOwner()) {
```

Comment:
For my own learning, what is an owner element? Does that mean items like `placeholder`, `overflow` etc? I.e. ones fully managed by their parent

Reply:
It applies to elements like amp-carousel that schedule resource loading for their children. Basically an opt-out of the Resources system.